### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by principles31**
**HTML/CSS & JavaScript capstone project - Conference page**

>  The goal of this project is to build is an online website for a conference

> Two pages are required to build
- Home page
- About page
-  Below is a screenshot of the home page 
- [Screenshot from 2022-04-30 08-09-46](https://user-images.githubusercontent.com/90258833/166097860-c164faad-fc49-49b2-9d1d-2930817b591b.png)
- About page
- [Screenshot from 2022-04-30 08-10-22](https://user-images.githubusercontent.com/90258833/166097888-6e8ec7da-732c-4ca2-b138-1225f6f57252.png)

> Each of these pages should have versions for 2 different screen sizes:
- Mobile: up to 768px wide.
- Desktop: 768px or wider..
- Below are screenshots of the mobile page for both
- 
- [Screenshot from 2022-04-29 20-58-40](https://user-images.githubusercontent.com/90258833/166068567-65afc80f-8b47-4d2a-b565-01ed08a31ffa.png)
- [Screenshot from 2022-04-29 20-59-46](https://user-images.githubusercontent.com/90258833/166068684-5ec7fc43-2434-46f4-8ddf-63fba83cc096.png)
- Below is a video documentation link
-  https://www.loom.com/share/3ee0fc6b2f4749b4b66363894f451063
- A live demo link
- https://principles31.github.io/module1_capstone_project/
- The changes requested by the previous reviewer have been implemented.
> Interactions
. Links
- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.
. Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
- The mobile menu should have a close (X) button that closes the menu.
. Dynamic page
- The section "Featured speakers" should be created dynamically in JavaScript.
- You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -5,33 +5,31 @@
...
     <script src="https://kit.fontawesome.com/b331332bf9.js" crossorigin="anonymous"></script>
-    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Lato:wght@300&family=Poppins&family=Roboto&display=swap"
+    <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Lato:wght@300&family=Poppins&family=Roboto&display=swap"
         rel="stylesheet">
     <link rel="stylesheet" href="about.css">
     <title>Web Developer Conference</title>
 </head>
 <body>
     <header class="about">
```
Comment (arslanbisharat) [inline about.html:15]: - [x] Both the Home and About page should contain the same navbar and footer. 

**Home Page Navbar**

image

**About Page Navbar**

image

File: about.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+humbergur.addEventListener('click', () => {
+  listItems.forEach((item) => item.classList.remove('hidden'));
+  listItems.forEach((item) => item.classList.add('show'));
+
+  humbergur.classList.add('hidden');
+  menuList.classList.add('ul_list');
+  navbar.classList.add('expand');
+  closeMenu.classList.add('close_show');
+
+//  listItems.forEach((item) => item.addEventListener('click', resetMobileMenu));
```
Comment (arslanbisharat) [inline about.js]: - [x] Remove commented code. This makes code hard to read

File: main.js [hunk 3]
```diff
@@ -1,72 +1,72 @@
...
+  speakerInfor.innerText = `${speakersInforArray[i].details}`;
+  speakerDetails.appendChild(speakerInfor);
+  const speakerPara = document.createElement('p');
+  speakerPara.setAttribute('class', 'para_speaker');
+  speakerPara.innerText = `${speakersInforArray[i].work}`;
+  speakerDetails.appendChild(speakerPara);
+  spanSpeaker.appendChild(speakerDetails);
+  speakerCard.appendChild(spanSpeaker);
+
+  // speakerSection.appendChild(mainSpeakers);
```
Comment (arslanbisharat) [inline main.js]: Remove commented code. This makes code hard to read

File: about.html [hunk 4]
```diff
@@ -5,33 +5,31 @@
...
-                <li class="desk_menu_list1"><a href="#">Logout</a></li>
-         </ul>
-
+            <li class="desk_menu_list1"><a href="#"><i class="fa_brands fa_facebook-f"></i></a></li>
+            <li class="desk_menu-list1"><a href="#"><i class="fa_brands fa_twitter"></i></a></li>
+            <li class="desk_menu_list1"><a href="#">English</a></li>
+            <li class="desk_menu_list1"><a href="#">My Page</a></li>
+            <li class="desk_menu_list1"><a href="#">Logout</a></li>
+        </ul>
         <nav class="about_nav second_nav">
```
Comment (arslanbisharat) [inline about.html:25]: - [x] On the mobile screen it is hard to see the hamburger icon. Please make sure its background is transparent or the same color/pattern as the top section background.

image

Comment (arslanbisharat) [inline about.html:25]: - [x] On Desktop the Navbar should be divided in a top dark bar and a bottom white bar but you have a top grey bar and a dark bottom bar. 

**Requirement**: 

image

**Clone**: 
image

File: about.html [hunk 5]
```diff
@@ -5,33 +5,31 @@
...
-                <li class="desk_menu_list1"><a href="#">English</a></li>
-                <li class="desk_menu_list1"><a href="#">My Page</a></li>
-                <li class="desk_menu_list1"><a href="#">Logout</a></li>
-         </ul>
-
+            <li class="desk_menu_list1"><a href="#"><i class="fa_brands fa_facebook-f"></i></a></li>
+            <li class="desk_menu-list1"><a href="#"><i class="fa_brands fa_twitter"></i></a></li>
+            <li class="desk_menu_list1"><a href="#">English</a></li>
+            <li class="desk_menu_list1"><a href="#">My Page</a></li>
+            <li class="desk_menu_list1"><a href="#">Logout</a></li>
```
Comment (arslanbisharat) [inline about.html]: - [x] Social Media icon is missing. Please add that in the top nav bar.

File: about.html [hunk 6]
```diff
@@ -44,50 +42,73 @@ <h2 class="about_second_heading">
...
+    <footer class="footer">
+        <div class="logo_footer">
+            <img class="footer_img" src="img/images/Hintergrund-2-nmg.png" alt="">
+        </div>
+        <div class="rights_reserved">
+            <p class="rights">2022 Web Developer Conference some rights Reserved </p>
+            <p class="para_rights">Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto obcaecati quos
+                dolorem.</p>
+        </div>
+    </footer>
```
Comment (arslanbisharat) [inline about.html:113]: - [x] Add some padding around both of the icon and text as present in the design. 

**Clone**: 

image

File: index.html [hunk 7]
```diff
@@ -47,110 +43,115 @@
                 <h1 class="hello_head">Hello, we Welcome you All!!!</h1>
```
Comment (arslanbisharat) [inline index.html:45]: - [x] Please add proper styling here by adding gap between main body and header and correct alignment. 
**Clone:** 

image

**Original**: 
image

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 7 above)
Comment (principles31) [reply index.html:45]: @arslanbisharat Thank you for the feedback.
the above-requested changes have been implemented and updated.

**Review by KDlamini — APPROVED** [review]
### Hi @principles31 🙋‍♀️,  

Great work!
You have made a great effort in implementing the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Great PR title
✔️ Good PR description
✔️ Good commits
✔️ No linters errors

# Status: Approved ✔️ 

giphyapproved

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@KDlamini Thank you for the feedback and your approval.

**Pull request merged**